localisation/00_zuwc_united_localisation.csv

```diff
@@ -1209,12 +1209,11 @@ EVTDESC_WCTHW_2010;The people who have become infected with the plague cannot be
-EVTDESC_WCTHW_2011_arthas;The news of your deeds has reached the courts. The Scourge drives people to carry out atrocious, but necessary, acts... You and your army have killed all of the infected residents of [From.GetName]. Among the infected people were men, women and children, but you showed no mercy. After the butchering, you had set the city on fire and left it in ruins to prevent the dead from rising once again.;;Die Nachricht von Ihren Taten hat die Hof erreicht.
- Die Gei�el bringt die Menschen dazu, schreckliche aber notwendig Dinge zu tun.\nDu und deine Armee haben die infizierten Einwohner [From.GetName] abgeschlachtet. Unter ihnen waren M�nner, Frauen und Kinder, aber du zeigte kein Erbarmen. Nach dieser schrecklichen Tat setzte du die Stadt in Brand und legte sie in Schutt und Asche um zu verhindern, dass die Toten wieder auferstehen.;;;;;;;;;;;x
+EVTDESC_WCTHW_2011_world;Terrible news has reached your court. The Scourge has driven even the most noble people to carry out atrocious acts... [Root.GetTitledFirstName] and [Root.GetHerHis] army have killed all of the infected residents of [From.GetName]. Among the infected people were men, women, and children, but [Root.GetFirstName] showed them no mercy. Following the butchering, [Root.GetSheHe] had [Root.GetHerHis] men set the city on fire, leaving it in ruins.;;Schreckliche Nachrichten haben Euren Hof erreicht. Die Gei?el bringt die Menschen dazu, schreckliche Dinge zu tun.\n[Root.GetTitledFirstName] und [Root.GetHerHis]e Armee haben die infizierten Einwohner [From.GetName] abgeschlachtet. Unter ihnen waren M?nner, Frauen und Kinder, aber [Root.GetFirstName] zeigte kein Erbarmen. Nach [Root.GetHerHis]er schrecklichen Tat setzte [Root.GetSheHe] die Stadt in Brand und legte sie in Schutt und Asche.;;;;;;;;;;;x
+EVTDESC_WCTHW_2011_arthas;Word of your deeds has spread far and wide to foreign courts. The Scourge has driven people such as you to carry out atrocious, but necessary, acts... You and your army have killed all of the infected residents of [From.GetName]. Among the infected people were men, women, and even children, but you showed them no mercy. Following the butchering, you had your men set the city on fire, leaving it in ruins to prevent the dead from ever rising again.;;Die Nachricht von Ihren Taten hat die Hof erreicht. Die Gei�el bringt die Menschen dazu, schreckliche aber notwendig Dinge zu tun.\nDu und deine Armee haben die infizierten Einwohner [From.GetName] abgeschlachtet. Unter ihnen waren M�nner, Frauen und Kinder, aber du zeigte kein Erbarmen. Nach dieser schrecklichen Tat setzte du die Stadt in Brand und legte sie in Schutt und Asche um zu verhindern, dass die Toten wieder auferstehen.;;;;;;;;;;;x
```

Comment:
Please change german to "Die Nachricht von Euren Taten hat den Hof erreicht. Die Geißel bringt die Menschen dazu, schreckliche aber notwendig Dinge zu verrichten.\nIhr und Eure Armee habt die infizierten Einwohner von [From.GetName] abgeschlachtet. Unter ihnen waren Männer, Frauen und Kinder, aber Ihr zeigtet kein Erbarmen. Nach dieser schrecklichen Tat setztet Ihr die Stadt in Brand und legtet sie in Schutt und Asche um zu verhindern, dass die Toten wieder auferstehen."

Reply:
Roger.